pennylane/noise/__init__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains the functionality to work with noise in PennyLane.
+
+.. currentmodule:: pennylane
+
+.. autosummary::
+    :toctree: api
+
+    partial_wires
+    wires_eq
+    wires_in
+    op_eq
```

Comment:
The code in [this example](https://xanaduai-pennylane--5674.com.readthedocs.build/en/5674/code/api/pennylane.noise.op_eq.html) does not work:
<img width="622" alt="Captura de pantalla 2024-05-13 a las 14 17 39" src="https://github.com/PennyLaneAI/pennylane/assets/65235481/0e3e7780-d876-4982-b3eb-7c5750ecf02f">

Reply:
Fixed! same as above.